test/backward/v2.0.5/Index.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto2";
```

Comment:
not a biggy but I think you copied this folder, right? what I did on the past was to create a gitsubmodule pointing to the

Reply:
That's 100x better than my hacky solution. I'll follow that pattern.